src/workloads/scale/LargeScaleLongLived.yml

```diff
@@ -0,0 +1,82 @@
+SchemaVersion: 2018-07-01
+Owner: Storage Engines
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 400
+
+Actors:
+- Name: LongLivedCreator
+  Type: Loader
+  Threads: 10
+  Phases:
+  - Repeat: 1
+    Database: &DB longlived
+    CollectionCount: &CollectionCount 1000
+    Threads: 10
```

Comment:
Should threads be here in the phase and in the higher level actor level?

Reply:
Oh yeah -- this is a bug/limitation of the loader if I remember correctly. Not sure if we fixed or not. I'll leave that for @guoyr to address.